Sources/Models/BlogPosts/BlogPost0004_OvertureSetters.swift

```diff
@@ -20,7 +20,7 @@ transformations out of smaller units.
 
 > This week's [episode](https://www.pointfree.co/episodes/ep15-setters-ergonomics-performance) explored providing a friendlier API to functional setters, and improved
 their performance by leveraging Swift's value mutation semantics. To make these ideas accessible to everyone
-we have updated our [Swift Overture](\(gitHubUrl(to: .repo(.overture)))) library to add these
+we have updated our [Swift Overture](https://www.github.com/pointfreeco/swift-overture) library to add these
```

Comment:
I think "www" redirects, but we may want to use plain ole "github.com".

Reply:
true true, done